compiler-utils/src/main/java/com/squareup/anvil/compiler/internal/PsiUtils.kt

```diff
@@ -91,62 +92,85 @@ public fun KtAnnotated.requireAnnotation(
 }
 
 @ExperimentalAnvilApi
+public fun KtAnnotated.requireAnnotations(
+  fqName: FqName,
+  module: ModuleDescriptor
+): List<KtAnnotationEntry> {
+  return findAnnotations(fqName, module)
+    .ifEmpty { null }
+    ?: throw AnvilCompilationException(
+      element = this,
+      message = "Couldn't find the annotation $fqName."
+    )
+}
+
+@ExperimentalAnvilApi
+@Deprecated("Repeatable")
 public fun KtAnnotated.findAnnotation(
   fqName: FqName,
   module: ModuleDescriptor
-): KtAnnotationEntry? {
+): KtAnnotationEntry? = findAnnotations(fqName, module).singleOrEmpty()
+
+@ExperimentalAnvilApi
+public fun KtAnnotated.findAnnotations(
+  fqName: FqName,
+  module: ModuleDescriptor
+): List<KtAnnotationEntry> {
   val annotationEntries = annotationEntries
```

Comment:
Doesn't look like it.

Reply:
It's an improvement, because I don't want to call the getter function multiple times. That's unnecessary overhead, because it might create a new collection every time.